tools/depth-quality/rs-depth-quality.cpp

```diff
@@ -102,9 +119,17 @@ int main(int argc, const char * argv[]) try
 
         if (!plane_fit) return;
 
-        const float bf_factor = baseline_mm * focal_length_pixels * TO_METERS; // also convert point units from mm to meter
+        const float bf_factor = baseline_mm * intrin->fx * TO_METERS; // also convert point units from mm to meter
 
-        std::vector<rs2::float3> points_set = points;
+        std::vector<rs2::float3> points_set;
```

Comment:
Please change the name of this vector to be more informative - for example deprojected_points